runtime-config.md

```diff
@@ -36,6 +36,15 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
     }
 }
 ```
+Note: The following filesystems MUST be made available in each Linux application's filesystem
```

Comment:
This is Linux-specific stuff, so I think it should go in the Linux-specific docs too.